### ISSUE TITLE
Document how to setup a repo

### ISSUE THREAD

**Issue by ryanmrichard** [NWChemEx-Project/DeveloperTools#81]
We should write down somewhere how and why our repos are structured the way they are. May also make sense to put some of these details in SDE so people using the SDE can follow suite.

**Comment by twindus** [comment]
@ryanmrichard I am guessing this is still a TODO.  Would some of the details go into PluginPlay now instead of SDE?

**Comment by ryanmrichard** [comment]
Yeah, this is still a TODO. I changed the original comment to SimDE (from SDE). I think the original sentiment was just in case someone was going to use (what's now PluginPlay) for non-chemistry purposes. I think wanting to use SimDE is a more realistic use case (that's what I do for example in GhostFragment).
